src/foundry/foundry.js/applications/formApplications/defaultTokenConfig.d.ts

```diff
@@ -30,11 +30,13 @@ declare global {
      * })
      * ```
      */
-    static get defaultOptions(): typeof FormApplication['defaultOptions'];
+    static override get defaultOptions(): typeof FormApplication['defaultOptions'];
 
-    override getData(options: unknown): Promise<DefaultTokenConfig.Data>;
+    override getData(options: unknown): Promise<Data>;
```

Comment:
```suggestion
    override getData(options: unknown): Data | Promise<Data>;
```
to allow deriving classes to do it async or not.